commands/events/addEvent.js

```diff
@@ -86,16 +86,9 @@ module.exports = {
     await mongo().then(async (mongoose) => {
       try {
         const event = new Event(eventObj);
-
-        await event
-          .save()
-          .then((doc) => {
-            str = "Event Successfully Added! 😀";
-          })
-          .catch((err) => {
-            str = "There was some error. 🙁";
-            console.log(err);
-          });
+        await event.save().then((doc) => {
+          str = "Event Successfully Added! 😀";
+        });
```

Comment:
No catching errors here?
If some error occurs, bot will try sending empty string as message to the channel.

Reply:
If some error occurs then the _try_ execution will stop then and there and the control will flow to the _catch_ [here](https://github.com/Knuth-Programming-Hub/KLE-bot/blob/discord-server/bot.js#L103).
However, I have made a small change here in this [commit](https://github.com/Knuth-Programming-Hub/KLE-bot/pull/23/commits/7476b95232b40ebb21016f3800c16c852ad398bc). Please review this now. 😅